app/system/libs/non-gregorian-calendar-picker/jquery-3.6.0.js

```diff
@@ -0,0 +1,4007 @@
+/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.org/license */
```

Comment:
Do the non gregorian pickers not work with the current jquery 3.4.1 installed in https://github.com/odk-x/app-designer/blob/master/app/system/libs/jquery-3.4.1.js? If they don't, I think it might be worth considering upgrading that one instead of adding a new one just for the non gregorian calendar pickers

Although now I'm not sure which is the one loaded on the ODK X system because I also see https://github.com/odk-x/app-designer/blob/master/app/config/assets/libs/jquery-3.4.1.js 🤔

Reply:
Yes, it does work with jQuery 3.4.1. 

I implemented the non-Gregorian calendar pickers on a webpage built with HTML, CSS, and JS as well as used the [jQuery file](https://github.com/odk-x/app-designer/blob/master/app/system/libs/jquery-3.4.1.js) loaded on the ODK-X System instead of the new one.

I added it just in case but can omit it in the new PR.